MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -43,15 +43,16 @@ public void Dispose()
             if (MediaQueue.HasNext())
             {
                 await CurrentPlaybackManager.Stop();
-                var item = MediaQueue[MediaQueue.Index + 1];
                 MediaQueue.SetNextAsCurrent();
-                await CurrentPlaybackManager.Play(item);
+                await Task.WhenAll(
+                    CurrentPlaybackManager.Play(_currentMediaFile),
+                    GetMediaInformation(new[] { _currentMediaFile }));
             }
             else
             {
                 // If you don't have a next song in the queue, stop and show the meta-data of the first song.
                 await CurrentPlaybackManager.Stop();
-                var item = MediaQueue[0];
+                //var item = MediaQueue[0];
```

Comment:
I would suggest removing the comments if they're not supposed to be in the code.